vm/src/pyobject.rs

```diff
@@ -927,6 +921,22 @@ pub trait ItemProtocol {
         vm: &VirtualMachine,
     ) -> PyResult;
     fn del_item<T: IntoPyObject>(&self, key: T, vm: &VirtualMachine) -> PyResult;
+    fn get_item_option<T: IntoPyObject>(
```

Comment:
Not a great fan of this method name. But fine for now.

Reply:
Me neither. Any better suggestions?

Reply:
`maybe_get_option`?